package.json

```diff
@@ -6,11 +6,11 @@
   "main": "index.js",
   "scripts": {
     "fix": "npm-run-all --continue-on-error \"fix:*\"",
-    "fix:markdownlint": "markdownlint --fix \"**/*\"",
+    "fix:markdownlint": "markdownlint --dot --fix \"**/*\"",
     "fix:prettier": "prettier --write \"**/*\"",
     "fix:yarn-deduplicate": "yarn install && yarn-deduplicate --strategy=fewer && yarn install",
     "lint": "npm-run-all --continue-on-error \"lint:*\"",
-    "lint:markdownlint": "markdownlint \"**/*\"",
+    "lint:markdownlint": "markdownlint --dot \"**/*\"",
```

Comment:
Bonus: add `--dot` to `markdownlint` calls

```
markdownlint --help

  Options:
  ...
  -d, --dot                                   include files/folders with a dot (for example `.github`)
  ...
```

We’ll need this for `.github/pull_request_template.md`. I should have added this option in #147, but I did not know the option was necessary.